collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -114,6 +121,16 @@ def gpu_charts(gpu):
             'options': [None, 'Memory Used by Each Process', 'MiB', fam, 'nvidia_smi.processes_mem', 'stacked'],
             'lines': []
         },
+        USER_MEM: {
+            'options': [None, 'Memory Used by Each User', 'MiB', fam, 'nvidia_smi.user_mem', 'stacked'],
+            'lines': []
+        },
+        USER_NUM: {
+            'options': [None, 'Number of User on GPU', 'Nr', fam, 'nvidia_smi.user_num', 'stacked'],
+            'lines': [
+                ['nr', 'nr'],
```

Comment:
What is this `['nr', 'nr']` dimension? From what i understand we add here only `user_num` dimension.